### ISSUE TITLE
Strategy for merging to core

### ISSUE THREAD

**Issue by NateEag** [josteink/emacs-oob-reboot#27]
I stumbled across this project [in this Reddit thread](https://www.reddit.com/r/emacs/comments/69dvkm/help_us_improve_the_default_emacs_outofthebox/) and thought I'd make a suggestion it brought to mind.

Rather than trying to change a ton of defaults up front, which could meet with a lot of resistance, perhaps a better way to start is to:

* Create a function that sets the newbie-friendly defaults as settled on by this project
* Get that function merged into core
* Add a front-and-center call to action button on the start screen saying "New to Emacs? Activate `modern-mode`." (or something along those lines)

I suspect that would be just about as good for giving newbies a comfortable environment, and would be far less troubling to current Emacsers.

Once that behavior is in core, over time, perhaps `modern-mode` could become a default (or perhaps just some of the settings in it).

Just a suggestion. If it's helpful, great - if not, no worries. Just figured I'd throw it out there.

**Comment by dchrzanowski** [comment]
That indeed does sound like a good idea. A lot of software these days has options like that, there is no reason why Emacs shouldn't right?

**Comment by pkkm** [comment]
I agree that it should be easy to continue to use traditional settings, but I think that turning modern settings into yet another option/mode is the wrong way to go. The whole point of defaults is for users to have good settings without needing to change anything.

Instead, I think modern settings should be the default and there should be a function that reverts them to their old values, e.g. `(traditional-mode)` or `(use-settings-from-emacs-version "25")`. This way, new users would get a pleasant experience out of the box, and existing users who didn't like the changes would just need to add a line at the top of their `init.el`.

**Comment by NateEag** [comment]
@pkkm That approach might be better.

The approach I've described is compatible with what you want, as I suggested in my first post. What you're suggesting can be viewed as the fourth step in my proposed approach:

* Make `modern-mode` (or whatever name it receives) a default that must be suppressed manually.

I believe you'll find it much easier to achieve that bullet point if you begin with the three I suggested, rather than aiming for it as the starting point.

I could be wrong.
